### DISCUSSION TITLE
Appium2: chromewebdriver

### DISCUSSION THREAD

**Discussion by gosr** [appium/appium#16862] [Q&A] [answered]
In Appium1, the chromewebdriver should be installed here:
`/Users/<YourUserName>/.nvm/versions/node/<SomeNodeVersion>/lib/node_modules/appium/node_modules/appium-chromedriver/chromedriver/mac`

However, I can't find any documentation where to place the driver when using Appium2.
Or should it be installed via CLI? (even though I can't see it when running `appium plugin list` or `appium driver list`)

**Comment by gosr** [comment, marked as answer]
I just found out that it's living inside the appium-espresso-driver folder here:
`/Users/<YourUserName>/.appium/node_modules/appium-espresso-driver/node_modules/appium-chromedriver/chromedriver/mac`.

**Comment by jlipps** [comment]
What are you trying to accomplish?

**Reply by gosr** [reply]
Test webview content.
Btw I decided to use the `chromedriver_autodownload` feature, and it all works fine now.
